tooling/ast_fuzzer/src/program/func.rs

```diff
@@ -598,28 +601,59 @@ impl<'a> FunctionContext<'a> {
             return self.gen_for(u);
         }
 
+        if freq.enabled_when("loop", self.budget > 1 && self.unconstrained()) {
+            return self.gen_loop(u);
+        }
+
+        if freq.enabled_when("while", self.budget > 1 && self.unconstrained()) {
+            return self.gen_while(u);
+        }
+
+        if freq.enabled_when("break", self.in_loop) {
+            return Ok(Expression::Break);
+        }
+
+        if freq.enabled_when("continue", self.in_loop) {
+            return Ok(Expression::Continue);
+        }
+
         if freq.enabled_when("call", self.budget > 0) {
             if let Some(e) = self.gen_call(u, &Type::Unit, self.max_depth())? {
                 return Ok(e);
             }
         }
 
+        if freq.enabled("drop") {
```

Comment:
This is a duplicate, I moved it up to the top earlier so it doesn't gain weight if other stuff is disabled, because then it starts dropping everything.